repos/fdbt-site/src/pages/api/pricingPerDistance.ts

```diff
@@ -146,6 +152,32 @@ export default (req: NextApiRequestWithSession, res: NextApiResponse): void => {
         return;
     }
 
+    const ticket = getSessionAttribute(req, MATCHING_JSON_ATTRIBUTE);
+    const matchingJsonMetaData = getSessionAttribute(req, MATCHING_JSON_META_DATA_ATTRIBUTE);
+
+    // edit mode
+    if (ticket && matchingJsonMetaData) {
+        const product = ticket.products[0];
+        const updatedProduct = { ...product, pricingByDistance: distanceCap };
+
+        // edit mode
+        const updatedTicket: WithIds<Ticket> = {
+            ...ticket,
+            products: [updatedProduct],
```

Comment:
```suggestion
            products: [{ ...product, pricingByDistance: distanceCap }],
```

Then remove updatedProduct

Reply:
I think this already happens on line 161 @alainajaffer

Reply:
was thinking it would be better as a one liner